pyspeckit/spectrum/units.py

```diff
@@ -968,6 +834,27 @@ def _update_from(self, obj):
                 'velocity_convention', 'wcshead', 'xtype'):
             self.__dict__[attr] = obj.__dict__[attr]
 
+    def find_equivalencies(self, velocity_convention, center_frequency, center_frequency_unit, equivalencies):
+        """
+        Utility function used to validate parameters and add equivalencies
+        """
+        if velocity_convention and center_frequency and equivalencies:
+            raise ValueError('You cannot specify all 3: velocity_convetion, center_frequency and equivalencies') 
```

Comment:
So this isn't quite right: the `velocity_convention` must be specified unless `equivalencies` is.  But, `center_frequency` and `equivalencies` can't both be specified.  And similarly, if `center_frequency` has a `unit` already, then `center_frequency_unit` should not be allowed.